src/android/OpenTokAndroidPlugin.java

```diff
@@ -383,6 +383,13 @@ public void onVideoEnabled(SubscriberKit arg0, String arg1) {
             // TODO Auto-generated method stub
 
         }
+
+        public void unsubscribe(JSONArray args) {
```

Comment:
We can use the `removeStreamView()` method which is almost identical to the `unsubscribe()` method that you created.